index.js

```diff
@@ -13,7 +13,7 @@ const { GitHub, context } = require("@actions/github");
     const { owner, repo } = context.repo;
     const labels = core
       .getInput("labels")
-      .split("\n")
+      .split(/[\n,]/)
```

Comment:
Why do we need to support comma separated strings?

Reply:
Does adding this break support for doing something like

```
- uses: christianvuerings/add-labels@v1
   with:
   labels: |
      hello
      world
   env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
I explained that in the description 😛 

It's so we can dynamically generate the `labels` list. I tried to make the output `Regression\nCode complete` and it added 3 labels - `Regression`, `Code complete`, and `Regression\nCode Complete`

<img width="846" alt="image" src="https://user-images.githubusercontent.com/6806697/126497746-1eff6b25-87de-47cd-99e5-06108086a213.png">

https://github.com/scorebet/sportsbook-ios/pull/2909

Reply:
Old behaviour is unchanged from my tests